virttest/qemu_devices/qcontainer.py

```diff
@@ -2547,6 +2547,8 @@ def define_hbas(
                 protocol_cls = qdevices.QBlockdevProtocolFTP
             elif filename.startswith("vdpa:"):
                 protocol_cls = qdevices.QBlockdevProtocolVirtioBlkVhostVdpa
+            elif "vhost-user-blk" in filename:
+                protocol_cls = qdevices.QBlockdevProtocolVirtioBlkVhostUser
```

Comment:
Have a question here:
for vdpa "filename=/dev/vhost-vdpa-x" is defined by the system automatically as a device after loading the vhost-vdpa module.

But for virtio-blk-vhost-user, filename was defined in qsd, like:  --export type=vhost-user-blk,id=export1,addr.type=unix,addr.path=**/tmp/vhost-user-blk1.sock**,node-name=fmt1 \

So if I set the socket name to addr.path=**/tmp/test.sock**, then no filename matched here, so no QBlockdevProtocolVirtioBlkVhostUser node created.

Reply:
Yes, I see， it's just a draft. Currently it is only supported in this way. (path=/tmp/vhost-user-blk1.sock, path=/tmp/vhost-user-blk2.sock)
For the sake of program readability, I suggest that we fix the naming format. We can discuss it later.